sql_generators/active_users_aggregates/templates/mobile_query.sql

```diff
@@ -0,0 +1,232 @@
+--- Query generated via sql_generators.active_users_aggregates.
+WITH baseline AS (
+  SELECT
+    submission_date,
+    normalized_channel,
+    client_id,
+    sample_id,
+    days_since_seen,
+    days_seen_bits,
+    days_created_profile_bits,
+    durations,
+    normalized_os,
+    normalized_os_version,
+    locale,
+    city,
+    country,
+    app_display_version,
+    device_model,
+    first_seen_date,
+    submission_date = first_seen_date AS is_new_profile,
+    uri_count,
+    is_default_browser,
+    CAST(NULL AS string) AS distribution_id,
```

Comment:
For `Focus Android`, the original aua [query](https://github.com/mozilla/bigquery-etl/blob/d590bdd401b5787e583486af52b83db46b983886/sql/moz-fx-data-shared-prod/telemetry_derived/unified_metrics_v1/query.sql#L106) did not have CAST(NULL AS string) for `distribution_id` but do have it for `isp`.  I wonder what is the difference?

Reply:
Related to querying both Legacy and Glean. I generated a separate template for this app until the move to Glean is complete.